inline_scan.sh

```diff
@@ -115,7 +118,7 @@ main() {
 
 get_and_validate_analyzer_options() {
     #Parse options
-    while getopts ':k:s:r:u:p:a:d:f:i:m:t:PgVh' option; do
+    while getopts ':k:s:u:p:a:d:f:i:m:t:PgVRh' option; do
```

Comment:
Options `u`, `p`, `t`, `g` don't have associated cases. Is it intentional? Also, `p_flag` (lowercase `p`) is referenced on line 210, `t_flag` on line 251, `g_flag` on 293, and `v_flag` (lowercase `v`) a few times.

Reply:
That's right, we want to leave this intentionally to the original mapping as much as possible for future updates. Also to incorporate vuln scans and others if needed.